app/components/settings/area-detail/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import {
+  Alert,
+  View,
+  Image
+} from 'react-native';
+import tracker from 'helpers/googleAnalytics';
+
+import I18n from 'locales';
+import LeftBtn from 'components/common/header/left-btn';
+import Theme from 'config/theme';
+import headerStyles from 'components/common/header/styles';
+import ActionButton from 'components/common/action-button';
+import styles from './styles';
+
+class AreaDetail extends Component {
+  componentDidMount() {
+    tracker.trackScreenView('AreaDetail');
+  }
+
+  handleDeleteArea = () => {
+    if (this.props.isConnected) {
+      this.props.deleteArea(this.props.area.id);
+      this.props.navigate('Settings');
+    } else {
+      Alert.alert(
+        I18n.t('commonText.connectionRequiredTitle'),
+        I18n.t('commonText.connectionRequired'),
+        [{ text: 'OK' }],
+        { cancelable: false }
+      );
+    }
+  }
+
+  render() {
+    const imageUrl = this.props.imageUrl !== undefined ? this.props.imageUrl : null;
```

Comment:
We were trying to think about a case in which we would not have an image after the image API fix but we couldn't find any so I am not going to add the placeholder image right now